swingset/src/main/java/com/nqadmin/swingset/SSBaseComboBox.java

```diff
@@ -913,7 +913,12 @@ private void updateRowset() {
 			//       setBoundColumnText or RowSetOps.updateColumnText
 
 			String tStringMapping = String.valueOf(mapping);
-			if (!getBoundColumnText().equals(tStringMapping)) {
+			String rowsetValue = getBoundColumnText();
+			if (rowsetValue==null) {
+				if (tStringMapping!=null) {
+					setBoundColumnText(tStringMapping);
+				}
+			} else if (!rowsetValue.equals(tStringMapping)) {
 				setBoundColumnText(tStringMapping);
 			}
 		}
```

Comment:
Looks like
```
if (!Objects.equals(getBoundColumnText(), tStringMapping)) {
```
would work. (but these nested ifs always get to me, so verify...)

Reply:
That looks like it would be cleaner. Wasn't familiar with Objects.equal().